ui/component/autoplayCountdown/view.jsx

```diff
@@ -155,7 +160,11 @@ function AutoplayCountdown(props: Props) {
               iconRight={skipPaid ? ICONS.WALLET : ICONS.REPLAY}
               onClick={() => {
                 setTimerCanceled(true);
-                doReplay();
+                if (skipPaid) {
+                  doOpenModal(MODALS.AFFIRM_PURCHASE, { uri, cancelCb: () => setTimerCanceled(false) });
+                } else {
+                  doReplay();
+                }
```

Comment:
Cancelling the autoplay countdown of a paid item causing the playlist play button to be stuck

- Open http://localhost:9090/@eevblog:7/eevblog-1396-blackmagic-atem-mini:d?lid=acd1afb0e8a16ba534d523a3ec49f0b28ae95bc0 in incognito
- Click "Next", then cancel the autoplay countdown.
- Click on the 3rd video.  It is stuck in the Big Red Button mode.